CRM/Contact/BAO/Relationship.php

```diff
@@ -51,10 +51,14 @@ class CRM_Contact_BAO_Relationship extends CRM_Contact_DAO_Relationship {
    * @throws \CRM_Core_Exception
    */
   public static function create(&$params) {
-    $params = self::loadExistingRelationshipDetails($params);
-    if (self::checkDuplicateRelationship($params, $params['contact_id_a'], $params['contact_id_b'], CRM_Utils_Array::value('id', $params, 0))) {
+
+    $extended_params = self::loadExistingRelationshipDetails($params);
```

Comment:
@mallezie can you please change this $extended_params to $extendedParams as CiviCRM follows camelCase conventions